src/schema/posts.ts

```diff
@@ -1101,6 +1219,52 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
   BaseContext
 >({
   Query: {
+    squadPostModerationsBySourceId: async (
+      _,
+      { sourceId }: { sourceId: string },
```

Comment:
From RFC we should have both `sourceID` and `status`, plus pagination mentioned already by Ante

```graphql
query SourceModerationPosts {
  sourcePendingPosts(
    sourceId: ID!, 
    status: 'pending',
    """
    Pagination
    """
    after: String,
    first: Int
    ): [Post]
}
```

Reply:
Should we make the `status` an array? Just so for regular members that posted something and we should pull both rejected and pending posts. Wdyt?

Reply:
Just make sure its de-dupped so if someone sends `['pending', 'pending', 'rejected']` it becomes `['pending', 'rejected']` and it only accepts valid enums.